examples/Nokia_LCD_Demo/Nokia_LCD_Font.ino

```diff
@@ -0,0 +1,78 @@
+#include <Nokia_LCD.h>
```

Comment:
As per Arduino convention (the file does not compile otherwise) a sketch file should have the same name as the directory. So I suggest changing the directory name to `Nokia_LCD_Font` to match the name of this sketch.

Reply:
I see, I did the change. Wise decision to add this validation :D